addons/account/tests/test_account_journal_dashboard.py

```diff
@@ -9,6 +9,10 @@
 @tagged('post_install', '-at_install')
 class TestAccountJournalDashboard(AccountTestInvoicingCommon):
 
+    def assertDashboardData(self, journal, expected_values):
+        dashboard_data = journal._get_journal_dashboard_data_batched()[journal.id]
+        self.assertDictEqual({**dashboard_data, **expected_values}, dashboard_data)
```

Comment:
`expected_values` still looks really duplicated. We can refactor them here

```suggestion
    def assertDashboardData(self, journal, sum_draft, sum_waiting, sum_late, **kwargs):
        expected_values = {
            'number_draft': 1,
            'sum_draft': format_amount(self.env, sum_draft, self.company_data['currency']),
            'number_waiting': 1,
            'sum_waiting': format_amount(self.env, sum_waiting, self.company_data['currency']),
            'number_late': 1,
            'sum_late': format_amount(self.env, sum_late, self.company_data['currency']),
            **kwargs,
        }
        dashboard_data = journal._get_journal_dashboard_data_batched()[journal.id]
        self.assertDictEqual({**dashboard_data, **expected_values}, dashboard_data)

Reply:
These are the values for the purchase/sales journals, but we may want to check other values for the bank/cash journals. And it might be restrictive if we want to use it to assert other values, what do you think?

Reply:
Let's rename the assert then, to something like `assertDashboardJournalPurchaseSale`.
Later when we want to add test for bank/cash we can think about it again